config/kubernetes/sig-release/teams.yaml

```diff
@@ -143,103 +143,26 @@ teams:
     - nikhita
     - spiffxp
     members:
```

Comment:
What's the pruning criteria applied here?  I'm surprised by some of the names being dropped versus no additions either.

Reply:
From discussion with @justaugustus : 

This list is the explicitly named individuals, plus the nested groups.  Anybody in a nested group doesn't need to be in the top level list, as they are already implicitly include.  Anybody who's not been recently active in the SIG gets removed periodically.  Anybody who's active in the SIG (and not a lead or other role in a nested list) will be added periodically.